contrib/ansible/deploy-devel-playbook.yml

```diff
@@ -16,6 +16,10 @@
     # Namespace to deploy CO to:
     cluster_operator_namespace: "openshift-cluster-operator"
   tasks:
+  - name: wait for apiserver and controller-manager deployment to finish
+    shell: |-
+      oc rollout status deployment/cluster-operator-apiserver -n {{ cluster_operator_namespace }}
+      oc rollout status deployment/cluster-operator-controller-manager -n {{ cluster_operator_namespace }}
```

Comment:
IIRC we're supposed to try to stick to command over shell as it's a bit safer particularly when you've got vars going in. (@abutcher correct me if I'm wrong)

Lets just do it as two separate tasks, more informative anyhow for someone watching it run.